src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -4,7 +4,7 @@ Ceph Dashboard Developer Documentation
 Note: The content of this file has been moved into the Ceph Developer Guide.
 
 If you're interested in helping with the development of the dashboard, please
-see ``/doc/dev/developer_guide/dash_devel.rst`` or the `online version
+see ``/doc/dev/developer_guide/dash-devel.rst`` or the `online version
```

Comment:
I have made this alteration in https://github.com/ceph/ceph/pull/63697.

I have not touched the other two files in this PR, because I don't have enough insight into their function.